src/call.c

```diff
@@ -1159,7 +1162,7 @@ int call_connect(struct call *call, const struct pl *paddr)
 		if (err)
 			return err;
 
-		call_set_mdir(call, call->estadir, call->estvdir);
+		call_set_media_direction(call, call->estadir, call->estvdir);
```

Comment:
returns err, but we are not interested in it.

Since `stream_set_ldir()` also returns void, I think `call_set_media_direction()` should also return void.

Reply:
Done.